### PR TITLE
WIP: Creating some basic CI script

### PR DESCRIPTION
**Pull request by bkochuna**
Just some commands to probe the CI environment

### CODE REVIEW

**Comment by bkochuna** [comment]
Needs review, then we can close.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+language: cpp
+os: linux
+
+script:
+  - echo $PWD && mkdir ../bld && cd ../bld && echo $PWD
```
Comment (Qicang) [inline .travis.yml:5]: Suggest to change bld to build for better understanding

**Review by Qicang — CHANGES_REQUESTED** [review]
Overall pretty good, just minor comments!

File: .travis.yml (hunk 1 above)
Comment (bkochuna) [reply .travis.yml]: Addressed in commit f827877

**Review by Qicang — APPROVED** [review]

**Pull request merged**